src/basekernels/rationalquad.jl

```diff
@@ -1,48 +1,62 @@
 """
-    RationalQuadraticKernel(; α = 2.0)
+    RationalQuadraticKernel(; α=2.0)
 
 The rational-quadratic kernel is a Mercer kernel given by the formula:
 ```
-    κ(x,y)=(1+||x−y||²/α)^(-α)
+    κ(x, y) = (1 + ||x − y||² / (2α))^(-α)
 ```
-where `α` is a shape parameter of the Euclidean distance. Check [`GammaRationalQuadraticKernel`](@ref) for a generalization.
+where `α` is a shape parameter of the Euclidean distance. Check
+[`GammaRationalQuadraticKernel`](@ref) for a generalization.
 """
 struct RationalQuadraticKernel{Tα<:Real} <: SimpleKernel
     α::Vector{Tα}
     function RationalQuadraticKernel(;alpha::T=2.0, α::T=alpha) where {T}
-        @check_args(RationalQuadraticKernel, α, α > zero(T), "α > 1")
+        @check_args(RationalQuadraticKernel, α, α > zero(T), "α > 0")
         return new{T}([α])
     end
 end
 
 @functor RationalQuadraticKernel
 
-kappa(κ::RationalQuadraticKernel, d²::T) where {T<:Real} = (one(T)+d²/first(κ.α))^(-first(κ.α))
+function kappa(κ::RationalQuadraticKernel, d²::T) where {T<:Real}
+    return (one(T) + d² / (2 * first(κ.α)))^(-first(κ.α))
+end
+
 metric(::RationalQuadraticKernel) = SqEuclidean()
 
-Base.show(io::IO, κ::RationalQuadraticKernel) = print(io, "Rational Quadratic Kernel (α = ", first(κ.α), ")")
+function Base.show(io::IO, κ::RationalQuadraticKernel)
+    print(io, "Rational Quadratic Kernel (α = $(first(κ.α)))")
+end
 
 """
-`GammaRationalQuadraticKernel([ρ=1.0[,α=2.0[,γ=2.0]]])`
+`GammaRationalQuadraticKernel([α=2.0 [, γ=2.0]])`
+
 The Gamma-rational-quadratic kernel is an isotropic Mercer kernel given by the formula:
 ```
-    κ(x,y)=(1+ρ^(2γ)||x−y||^(2γ)/α)^(-α)
+    κ(x, y) = (1 + ||x−y||^γ / α)^(-α)
 ```
 where `α` is a shape parameter of the Euclidean distance and `γ` is another shape parameter.
 """
 struct GammaRationalQuadraticKernel{Tα<:Real, Tγ<:Real} <: SimpleKernel
     α::Vector{Tα}
     γ::Vector{Tγ}
-    function GammaRationalQuadraticKernel(;alpha::Tα=2.0, gamma::Tγ=2.0, α::Tα=alpha, γ::Tγ=gamma) where {Tα<:Real, Tγ<:Real}
-        @check_args(GammaRationalQuadraticKernel, α, α > one(Tα), "α > 1")
-        @check_args(GammaRationalQuadraticKernel, γ, γ >= one(Tγ), "γ >= 1")
+    function GammaRationalQuadraticKernel(
+        ;alpha::Tα=2.0, gamma::Tγ=2.0, α::Tα=alpha, γ::Tγ=gamma,
```

Comment:
I guess
```suggestion
        ;alpha::Tα=2.0, gamma::Tγ=2, α::Tα=alpha, γ::Tγ=gamma,
```
would be more performant in the default case? The question here is also if we should use `alpha = 1` as
default value.

Reply:
Although probably, since we divide gamma by 2 anyway in the computation it doesn't matter.

Reply:
Hmmm I wonder whether this is what you want. Generally speaking, if you're using this kernel you probably don't want the `Integer` parameter value, otherwise you would be using the `RationalQuadraticKernel`. My opinion is that making this parameter a float by default probably does make sense as a consequence of this.

Reply:
Makes sense.